### PR TITLE
Add com.leinardi.gwe

### PR DESCRIPTION
**Pull request by leinardi**
GWE is a GTK system utility designed to provide information, control the fans and overclock your NVIDIA video card and graphics processor.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.leinardi.gwe
testbot, build com.leinardi.gwe

**Comment by leinardi** [comment]
> To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user http://repo-test.flathub.org:8080/build-repo/1543/com.leinardi.gwe.flatpakref
> ```

Tested and it seems to work fine.

File: libdazzle.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (barthalion) [inline libdazzle.json]: Is there a reason to keep this module separately?

**Comment by leinardi** [comment]
Hi, at the beginning I had all the modules inside the main manifest file, than @TingPing suggested to use a script to generate and maintain the python modules and after that all modules but libdazzle where external. Than I moved libdazzle too to a separate file, for consistency.

File: libdazzle.json (hunk 1 above)
Comment (leinardi) [reply libdazzle.json]: Hi, at the beginning I had all the modules inside the main manifest file, than @TingPing suggested to use a script to generate and maintain the python modules and after that all modules but libdazzle where external. Than I moved libdazzle too to a separate file, for consistency.

**Comment by nedrichards** [comment]
As an app it would be stricly *better* if it had a nicer experience if you load it when you're not on an NVidia system but it seems to work fine insamuch as I can test it.

**Comment by leinardi** [comment]
Sure, I have it in my TODO list. Currently I'm focusing on an MVP, as soon as I'm sure that I can easily distribute the app across multiple distributions, I'll focus on polishing it.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.leinardi.gwe

**Pull request closed without merging**